nabo/nabo.cpp

```diff
@@ -46,6 +46,17 @@ namespace Nabo
 {
 	using namespace std;
 
+  struct stringformat : private stringstream {
```

Comment:
`stringformat` seems a suboptimal name to me here. It is more a `stringbuilder` or similar. Yes it is `boost::format`, but that class is containing a formatting string as member data.